h2/src/main/org/h2/value/Value.java

```diff
@@ -534,13 +535,17 @@ public Value modulus(@SuppressWarnings("unused") Value v) {
         throw throwUnsupportedExceptionForType("%");
     }
 
+    public Value convertTo(int targetType) {
+        return convertTo(targetType, -1, null);
+    }
+
     /**
      * Compare a value to the specified type.
      *
      * @param targetType the type of the returned value
      * @return the converted value
      */
-    public Value convertTo(int targetType) {
+    public Value convertTo(int targetType, int precision, Mode mode) {
```

Comment:
please add @param tags, especially mentioning the two special values for precision